packages/block-editor/src/store/private-actions.js

```diff
@@ -360,3 +360,13 @@ export function stopEditingAsBlocks( clientId ) {
 		dispatch.__unstableSetTemporarilyEditingAsBlocks();
 	};
 }
+
+export function registerBlockBindingsSource( source ) {
+	return {
+		type: 'REGISTER_BLOCK_BINDINGS_SOURCE',
+		sourceName: source.name,
+		sourceLabel: source.label,
+		sourceComponent: source.component,
+		useSource: source.useSource,
+	};
+}
```

Comment:
Since this is private API, we don't need to worry about this `registerBlockBindingsSource()` being perfect or backwards compatible, right?

Reply:
That's it. Keeping it private, and only used in Gutenberg, makes it easier to change it in the future without worrying too much.